notebooks/220-cross-lingual-books-alignment/README.md

```diff
@@ -0,0 +1,27 @@
MzISIAESIAESIIFYCcQ+vqIBirXltqResXfQlsjIapIACZAACZAACQREIPbxFQ1QQI2NoXRPIPYO2j0RnkECJEACJEACJEAC1QjEPr6iAarWPnh2nwnE3kH7jJeXJwESIAESIAESaCGB2MdXNEAtbNQxVTn2DhqTVqwLCZAACZAACZBAMwjEPr6iAWpGO2SUKQRi76AUngRIgARIgARIgAR6TSD28RUNUK9bFK/nlUDsHdQrLBZGAiRAAiRAAiRAAgUIxD6+ogEq0Ah4SLgEYu+g4ZJnZCRAAiRAAiRAArESiH18RQMUa8ttSb1i76AtkZHVJAESIAESIAESCIhA7OMrGqCAGhtD6Z5A7B20eyI8gwRIgARIgARIgASqEYh9fEUDVK198Ow+E4i9g/YZLy9PAiRAAiRAAiTQQgKxj69ogFrYqGOqcuwdNCatWBcSIAESIAESIIFmEIh9fEUD1Ix2yChTCMTeQSk8CZAACZAACZAACfSaQOzjKxqgXrcoXs8rgdg7qFdYLIwESIAESIAESIAEChCIfXxFA1SgEfCQcAnE3kHDJc/ISIAESIAESIAEYiUQ+/iKBijWltuSesXeQVsiI6tJAiRAAiRAAiQQEIHYx1c0QAE1NobSPYHYO2j3RHgGCZAACZAACZAACVQjEPv4igaoWvvg2X0mEHsH7TNeXp4ESCBwAocOHTILFiwwBw8eNPPnz+9Eu2XLFvPaa6+ZI0eOmPHjx+fW4s6dO2bFihVm4cKF9tgDBw6YHTt2mKtXr5p58+bZvxUt6+zZs2bq1Kmda545c8ZMmTLF4O9S7vDhw3NjKnKAzzKFpVzXZZoVz82bN82yZcvMxo0bU3lfunTJfr9r1y4zcuTIItXjMSTQNwKxj69ogPrWtHhhHwRi76A+GLEMEiCBeAlg0L5//34zceJEs2nTJgNjgcE4zNC7775b2LRoAwSzIh+Uf/nyZbNu3bpCEHE8TA7+H4N8iUXO922ACgVV4CCYk+XLl5udO3daA4P/hvHbvXu3NW8+PjBrMKbCxkeZLIME6iIQ+/iKBqiulsNye0Jg+vTp5tSpUz25Fi9CAiRAAqERQJbiwoULZsSIEWbJkiXWdEhW5NatW52MhM7KbN68uWNoMCBfv369efzxx23VkMXAB0Zl7ty55lvf+pb9723btpk///nPnewFzhs3btyQrFOaiYIJunHjhv2fGKCPP/7YnnvixAlr3iS7JGXs3bvXXnfx4sU2E3Xs2DFz+vRpc+XKFXuO1EHqCoO1atUq8+CDD9pYdZliZsAJ5127ds2WqbNQeeZEOCEmZLQmTZpkM2ajR4+2mTbE++KLL5o1a9aYl19+2X6/Z88eqw2OnzBhQqe+iAFsYbDwvcSKY/ghgVAIfOMb3zB/+MMfQgnHexw0QN6RskASIAESIAES6A0BydDgajNmzLDZCgzWH374YXP48GFrgPCR7MaYMWPswH3atGlm7NixnYwEzIlkPMQAifFABgjmYsOGDWbRokVm1KhRidO93CyKS0DMytatW83AwICdbudOjYNZwueLX/yizR7JtLJz5851MkuIVeojpgoGCLGhTBgrMMAHcUt95e9JBgjHuiZHMj/aHOF6YLp9+3Z7PAwQri2xigFCeeB3/vz5DuOLFy92/g1zJAay2yxbb1oWr0ICcROgAYpbX9aOBEiABEggYgIyeIb5OXnypM0CrV271qxcudIaFgzWdeYFWQ9tmoAGA/i0NUDIvMgUOJyHz+TJk82+ffs6U+4Eb946GJ2tgVlBJkc+aVkg+TsMEDJAMBXIHokxcg0Q6gLjInVEpkVPbSuyZkgyRogNmSlc250GKLxgJGGsXAMk5k4zQawyBQ5mCOukdL0jbqasGgkER4AGKDhJGBAJkAAJkAAJFCMgA30xPo8++qh56623rBFyTYJM+yprgGAMYHweeughc9999w2Z/oZo06bAiemAGTl69Kg1XMjMJG0YIFP1ZNqYzgD1ygAJeZnmh//2bYBkEwTZeEGm+vnaHKJY6+FRJNBeAjRA7dWeNScBEiABEmg4AT19CgP2t99+2zz22GNm1qxZHQOEKladAiebGMBEvfnmmzb7k7S7XJFNEGQKnEwf01PMkBmRdUKYPrZ06dJOFqaMASo6Bc6NG8zAE5k1+TeOkel3P/vZz+xan7QMUFrdJAOEKXAyZbFIVqrhzZThk0BwBGiAgpOEAZEACZAACZBAMQLaAGEgLYbBXadTZhMEvQZIDFCRwbq7nXTSNthpmyDIrnGYHrd69Wpz+/Ztu64Hmx+UMUCIu8gmCGJysCGEfPQ22LpO+Pvs2bOHrC1yp8C9//771rjhI/WXOObMmWMzdEmbQBRTnUeRAAlUJUADVJUgzycBEiABEiCBlhBI2v2tSVWXzRGKbuvdbd3ctUHdns/jSYAEekOABqg3nHkVEiABEiABEmgsARnYIxPTpPfYuNtqz5w5s9b4aYAa28QZeMsI0AC1THBWlwRIgARIgARIgARIgATaTIAGqM3qs+4kQAIkQAIkQAIkQAIk0DICNEAtE5zVJQESIAESIAESIAESIIE2E6ABarP6rDsJkAAJkAAJkAAJkAAJtIwADVDLBGd1SYAESIAESIAESIAESKDNBGiA2qw+604CJEACJEACJEACJEACLSNAA9QywVldEiABEiABEiABEiABEmgzARqgNqvPupMACZAACZAACZAACZBAywjQALVMcFaXBEiABEiABEiABEiABNpMgAaozeqz7iRAAiRAAiRAAiRAAiTQMgI0QC0TnNUlARIgARIgARIgARIggTYToAFqs/qsOwmQAAmQAAmQAAmQAAm0jAANUMsEZ3VJgARIgARIgARIgARIoM0EaIDarD7rTgIkQAIkQAIkQAIkQAItI0AD1DLBWV0SIAESIAESIAESIAESaDMBGqA2q8+6kwAJkAAJkAAJkAAJkEDLCPwf95mxeO+Ew5QAAAAASUVORK5CYII=" />
```

Comment:
image in base64 format looks scary :)
can we replace it with usual image (maybe screenshot)

Reply:
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/51917466/18f3ab38-e264-4b2c-a088-8e54b855c1b2)

Reply:
<img width="568" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/51917466/3bb85143-627b-4f02-b628-7bef37823520">

Reply:
<img width="265" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/51917466/7e5cf756-fa4c-4427-84be-04326433cd9a">

Reply:
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/51917466/10967fb7-eef4-4074-981f-279dd66fef89)

Reply:
Done